influxdb-mixin/dashboards/influxdb-instance-overview.libsonnet

```diff
@@ -0,0 +1,1881 @@
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local activeQueriesPanel(matcher) = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(job, influxdb_cluster, instance) (qc_compiling_active{' + matcher + ', instance=~"$instance"})',
+      datasource=promDatasource,
+      legendFormat='{{instance}} - compiling',
+    ),
+    prometheus.target(
+      'sum by(job, influxdb_cluster, instance) (qc_queueing_active{' + matcher + ', instance=~"$instance"})',
+      datasource=promDatasource,
+      legendFormat='{{instance}} - queueing',
+    ),
+    prometheus.target(
+      'sum by(job, influxdb_cluster, instance) (qc_executing_active{' + matcher + ', instance=~"$instance"})',
+      datasource=promDatasource,
+      legendFormat='{{instance}} - executing',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Active queries',
+  description: 'Number of active queries for the server.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisBorderShow: false,
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 20,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'smooth',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'normal',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      decimals: 0,
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local httpOperationsPanel(matcher) = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(job, influxdb_cluster, instance, status) (rate(http_query_request_count{' + matcher + ', instance=~"$instance"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='{{instance}} - query - {{status}}',
+    ),
+    prometheus.target(
+      'sum by(job, influxdb_cluster, instance, status) (rate(http_write_request_count{' + matcher + ', instance=~"$instance"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='{{instance}} - write - {{status}}',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'HTTP operations',
```

Comment:
I'd suggest suppressing metrics with values = 0, as currently you have this situation where only half the listed metrics are relevant
![image](https://github.com/grafana/jsonnet-libs/assets/1282515/fed0da2f-276f-43ce-9204-ff1f6ffa7bec)

Reply:
Update: Saw this applied to multiple panels, I'd give that a sanity check throughout if possible.

Basically my fear here is that that's going to multiply with the number of instances, so if it's something like the IQL Query Response Time panel that would probably be fine since it's relevant to see 0s there, but for things like HTTP response it grows out of hand quite quickly.

![image](https://github.com/grafana/jsonnet-libs/assets/1282515/4362d85d-7d38-4f69-8d64-a0da686634e8)

Also think this is a good one to get @rgeyer's thoughts on before actioning

Reply:
That is a very good point re: usability on the metrics changing assigned colours, especially with multiple instances...

Ok yeah then lets exercise caution on this one and maybe just revisit it in the future when we've had time to discuss with Vitaly? Don't want to block the PR

Reply:
@Dasomeone @rgeyer Is there a setting in Grafana to suppress 0 values? Nothing I've tried is working quite how I'd like.

Reply:
Simply append `> 0` to the end of the query. Should be all you need, though the caveat is that it'll still show up if at any point in the time range it was above 0. Not the end of the world with a properly set time range of 30 minutes or so, but yeah :)